.goreleaser.yml

```diff
@@ -1,4 +1,7 @@
 project_name: helmfile
+env:
+  # We default to non-v1 mode (=helmfile v0.x behavior) when HELMFILE_V1MODE is not set
+  - HELMFILE_V1MODE={{ if index .Env "HELMFILE_V1MODE"  }}{{ .Env.HELMFILE_V1MODE }}{{ else }}false{{ end }}
```

Comment:
This is needed to avoid `{{.Env.HELMFILE_V1MODE}}` failing due to the missing map entry error when you run goreleaser without setting HELMFILE_V1MODE, which is the case when you run it locally without the envvar, or the case when goreleaser is invoked via the publish_binaries workflow.

Reply:
You can optionally do it directly like:

```
- -X github.com/helmfile/helmfile/pkg/runtime.v1Mode={{ if index .Env "HELMFILE_V1MODE" }}{{ .Env.HELMFILE_V1MODE }}{{ else }}false{{ end }}
```

I slightly prefer the current form for readability. WDYT?

Reply:
I think so.